queryable-encryption/node/aws/reader/your_credentials.js

```diff
@@ -9,7 +9,7 @@ function getCredentials() {
 const credentials = {
   // Mongo Paths + URI
   MONGODB_URI: "<your MongoDB URI here>",
-  SHARED_LIB_PATH: "<path to CSFLE shared library>",
+  SHARED_LIB_PATH: "<path to crypt_shared library>",
```

Comment:
The "CSFLE" -> "crypt_shared" update should have been included in a previous PR. Adding it here as I had to rebuild all sample applications.